src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/DateTimePicker.DateTimePickerAccessibleObjectTests.cs

```diff
@@ -240,5 +240,66 @@ public void DateTimePickerAccessibleObject_Collapse_IfHandleIsCreated_ReturnsExp
             Assert.Equal(UiaCore.ExpandCollapseState.Collapsed, accessibleObject.ExpandCollapseState);
             Assert.True(dateTimePicker.IsHandleCreated);
         }
+
+        public static IEnumerable<object[]> DateTimePickerAccessibleObject_DefaultAction_ReturnsExpected_TestData()
+        {
+            // Expanded dtp control has "Collapse" as a default action, else "Expand".
+            yield return new object[] { true, SR.AccessibleActionCollapse };
+            yield return new object[] { false, SR.AccessibleActionExpand };
+        }
+
+        [WinFormsTheory]
+        [MemberData(nameof(DateTimePickerAccessibleObject_DefaultAction_ReturnsExpected_TestData))]
+        public void DateTimePickerAccessibleObject_DefaultAction_ReturnsExpected(bool isExpanded, string expected)
+        {
+            using DateTimePicker dateTimePicker = new();
+
+            dateTimePicker.CreateControl();
+            var accessibleObject = (DateTimePickerAccessibleObject)dateTimePicker.AccessibilityObject;
```

Comment:
nit:
```suggestion
            using DateTimePicker dateTimePicker = new();
            dateTimePicker.CreateControl();
            
            var accessibleObject = (DateTimePickerAccessibleObject)dateTimePicker.AccessibilityObject;
```

Also, do we need `accessibleObject` variable and do we really need this cast (e.g. what are the advantages of having it)?

Reply:
We can remove the cast, because the base class has all methods to call. So this unnecessary cast has been deleted.